src/sentry/static/sentry/app/views/events/utils/eventsRequest.tsx

```diff
@@ -167,6 +168,10 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
      */
     showLoading: PropTypes.bool,
 
+    /**
+     * Name used for display current series data set tooltip
+     */
+    currentSeriesName: PropTypes.string,
```

Comment:
What about just `seriesName`?

Reply:
There seems to be a previous? What if we want to support that in the future? :D